sdk/cosmos/azure-cosmos/test/test_auto_scale_async.py

```diff
@@ -18,36 +18,37 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-import unittest
+
 from azure.cosmos.aio import CosmosClient
 import azure.cosmos.exceptions as exceptions
-from azure.cosmos import ThroughputProperties, PartitionKey, http_constants
+from azure.cosmos import ThroughputProperties, PartitionKey
 import pytest
 import test_config
 
 pytestmark = pytest.mark.cosmosEmulator
 
 
 @pytest.mark.usefixtures("teardown")
-class AutoScaleTest(unittest.TestCase):
+class TestAutoScaleAsync:
     host = test_config._test_config.host
     masterKey = test_config._test_config.masterKey
     connectionPolicy = test_config._test_config.connectionPolicy
 
     @classmethod
-    async def setUpClass(cls):
+    async def _set_up(cls):
         if (cls.masterKey == '[YOUR_KEY_HERE]' or
                 cls.host == '[YOUR_ENDPOINT_HERE]'):
             raise Exception(
                 "You must specify your Azure Cosmos account values for "
                 "'masterKey' and 'host' at the top of this class to run the "
                 "tests.")
 
-        cls.client = CosmosClient(cls.host, cls.masterKey, consistency_level="Session",
-                                  connection_policy=cls.connectionPolicy)
+        cls.client = CosmosClient(cls.host, cls.masterKey)
```

Comment:
We are not passing consistency level default value here, what will be used?

Reply:
The account default consistency will be used